helm/cfgov/values.yaml

```diff
@@ -179,6 +179,14 @@ cronJobs:
       - "clearsessions"
     restartPolicy: OnFailure
 
+  - name: "publishPages"
```

Comment:
This is such a minor thing, but I think we have an opportunity here to establish the naming pattern for these jobs that run a Django management command. I didn't think about this for `clearsessions`, but I think I'd prefer the job names to be the same as the Django management command, so `publish_scheduled_pages` in this case. I don't think there's any reason  not to do so?

We can rename `clearsessions` in another PR.

Reply:
Just needs to be FQDN valid. 


[Kubernetes Documentation](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/)
```
When creating the manifest for a CronJob resource, make sure the name you provide is a valid [DNS subdomain name](https://kubernetes.io/docs/concepts/overview/working-with-objects/names#dns-subdomain-names). The name must be no longer than 52 characters. This is because the CronJob controller will automatically append 11 characters to the job name provided and there is a constraint that the maximum length of a Job name is no more than 63 characters.
```

Reply:
Hmm, I wasn't aware of the DNS requirement. 

I think I'd still prefer kebab casing over camel case, since that's closer. I dislike though that the names aren't the same, but that gets us closer.

I think as long as we follow a particular name formatting convention (again, my preference is Django's, or as close as we can get, since this is a Django app), we can simply name our jobs following that if they use multiple management commands (or if they use no management commands at all). But for jobs that simply wrap a management command, I strongly prefer naming them as close to the management command as we can get.